homeassistant/components/permobil/sensor.py

```diff
@@ -202,28 +202,22 @@ async def async_setup_entry(
     )
 
 
-class PermobilSensor(CoordinatorEntity[MyPermobilCoordinator], SensorEntity):
+class PermobilSensor(PermobilEntity, SensorEntity):
     """Representation of a Sensor.
 
     This implements the common functions of all sensors.
     """
 
-    _attr_has_entity_name = True
     _attr_suggested_display_precision = 0
     entity_description: PermobilSensorEntityDescription
-    _available = True
 
     def __init__(
         self,
         coordinator: MyPermobilCoordinator,
         description: PermobilSensorEntityDescription,
     ) -> None:
         """Initialize the sensor."""
-        super().__init__(coordinator=coordinator)
-        self.entity_description = description
-        self._attr_unique_id = (
-            f"{coordinator.p_api.email}_{self.entity_description.key}"
-        )
+        super().__init__(coordinator=coordinator, description=description)
```

Comment:
```suggestion
        super().__init__(coordinator, description)
```

when the keys are the same as the value, it doesn't add much value imo